app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -1,5 +1,9 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
```

Comment:
this should probably be disabled locally with /* eslint-disable-next-line no-param-reassing */